packages/rich-text/src/plugins/List/index.tsx

```diff
@@ -157,12 +161,67 @@ const withCustomList = (options): WithOverride => {
 
     withDefaultOverrides(editor);
 
-    // Reverts any overrides to insertFragment
+    // // Reverts any overrides to insertFragment
     editor.insertFragment = insertFragment;
 
-    // Use our custom getListInsertFragment
+    // // Use our custom getListInsertFragment
     editor.insertFragment = getListInsertFragment(editor);
 
+    const { normalizeNode } = editor;
+    editor.normalizeNode = (entry) => {
+      const [node, path] = entry;
+
+      // If the element is a list, ensure its children are list nodes.
```

Comment:
What's our strategy to ideally account for our A/C's
> Try to encapsulate the code in a reusable way so that it can be used in other normalization callbacks
?

Can we e.g. use `rich-text-types` constants instead of hardcoding certain things here and extract it into a reusable function?